java/src/jmri/jmrix/AbstractThrottleManager.java

```diff
@@ -606,11 +596,14 @@ public boolean addressStillRequired(LocoAddress la) {
 
     @Override
     public void releaseThrottle(DccThrottle t, ThrottleListener l) {
+        log.debug("AbstractThrottleManager.releaseThrottle: {}, {}", t.toString(), l.toString());
         disposeThrottle(t, l);
     }
 
     @Override
     public boolean disposeThrottle(DccThrottle t, ThrottleListener l) {
+        log.debug("AbstractThrottleManager.disposeThrottle: {}, {}", t.toString(), l.toString());
```

Comment:
Remove the `toString()` calls in this log line. In addition to the above concerns about `toString()` and logging, this call here forces the ThrottleListener (`l`) passed to be non-null, while the rest of this method accepts `l` being null. Its this line that is causing all CI tests to fail.

Alternatively, annotate (using `@Nonnull`) the ThrottleListener parameter as requiring a non-null ThrottleListener. Note that at least the `jmri.jmrix.can.cbus.CbusThrottleManager` expects to be able to call this method with a null ThrottleListener (and a number of tests do as well, even if the tested classes normally do not use null ThrottleListeners).